src/cljs/main/broadfcui/main.cljs

```diff
@@ -112,6 +112,30 @@
                   "http://status.firecloud.org/"]
                  " for more information."])}))
 
+(defn status-alert [title message link]
+  [:div {:style {:border "1px solid" :borderColor "#bcaeac" :backgroundColor "#f7e4e1" :padding "1rem"}}
+   [:div {:style {:marginBottom "0.5rem"}} title]
+   [:div {:style {:color "#0a0a0a" :fontSize "80%"}}
+    [:div {} (str message " ")
+     [:a {:href (str link) :target "_blank" :style {:color "#1779ba"}} "Read more..."]]]])
+
+(react/defc ShowStatusAlerts
+  {:render
+   (fn [{:keys [this state]}]
+     (let [alerts (:alerts @state)]
+       (when (not (empty? alerts))
+         [:div {}
+          (map (fn [alert]
+                 (status-alert (:title alert) (:message alert) (:link alert)))
+               alerts)])))
+   :component-did-mount
+   (fn [{:keys [state]}]
+     (utils/ajax {:url "http://storage.googleapis.com/firecloud-alerts-dev/alerts.json"
```

Comment:
I have this in config but there's some sort of race condition where it's not loaded in time so I need to solve that before I use the config'd version